client/src/components/Auction.jsx

```diff
@@ -40,7 +37,7 @@ class Auction extends Component {
       dispatch(Auctions.fetchingAnAuction(false));
       dispatch(Auctions.fetchAuctionErrored(true, err));
     });
-    //lala:
+
     if (user.username) {
```

Comment:
I think we should try and be consistent with using ES6, especially now that we are closer to the end and need to polish up our codebase.